lib/parsers/document_parser.js

```diff
@@ -305,6 +303,26 @@ function fillAssociations() {
   }
 }
 
+function convertAnnotationsToOptions(annotations) {
+  const result = {};
+  annotations.forEach(o => {
```

Comment:
could you not use `o` as variable name here please?

Reply:
sure